Tests/test_PDB.py

```diff
@@ -1332,6 +1332,25 @@ def test_NACCESS_asa_file(self):
             naccess = process_asa_data(asa)
         self.assertEqual(len(naccess), 524)
 
+class TestSelf(unittest.TestCase):
```

Comment:
The fact that this was once a self test is not really of interest. Can you give this class and the test a more meaningful name? Something about PDB input/output?